modules/ROOT/pages/mule-high-availability-ha-clusters.adoc

```diff
@@ -13,103 +16,118 @@ Contact your customer service representative about pricing for this feature.
+* Salesforce streaming API. +
+If multiple instances of the same application are deployed, they will fail because the API only supports a single consumer. There is no failover support in case the instance connected is stopped or crashes.
 
+== Prerequisites
 
+* You must have at least two Mule runtime engines in a cluster, each one running on different machines.
+* Mule high availability (HA) requires a reliable network connection between servers to maintain synchronization between the nodes in the cluster.
+* You must keep the following ports open to set up a Mule cluster: port `5701` and port `54327`.
+* In case you configure your cluster to use multicast, you need to enable the multicast IP address: `224.2.2.3`.
++
+Multicast is used to discover new cluster members.
 
-== About Clustering
+== Clustering
 
-With out of the box configurations, it is recommended that you scale a cluster to no more than eight Mule runtimes. All the Mule runtimes in a cluster group together to form a single unit. Thus, you can deploy, monitor, or stop all the Mule runtimes in a cluster as if they were a single Mule runtime. If you need more nodes than eight, contact https://www.mulesoft.com/support-and-services/mule-esb-support-license-subscription[MuleSoft Support].
+With the initial configuration, MuleSoft recommends that you scale a cluster to no more than eight Mule runtime engines. All the Mule runtime engines in a cluster group together to form a single unit. Thus, you can deploy, monitor, or stop all the Mule runtime engines in a cluster as if they were a single Mule runtime engine.
 
-All the Mule runtimes in a cluster share memory, as illustrated below:
+All the Mule runtime engines in a cluster share memory, as illustrated below:
 
 image::topology-4-cluster.png[topology_4-cluster]
 
-Mule uses an *active-active* model to cluster Mule runtimes, rather than an *active-passive* model.
-
-In an *active-passive* model, one node in a cluster acts as the *primary*, or active node, while the others are *secondary*, or passive nodes. The application in such a model runs on the primary node, and only ever runs on the secondary node if the first one fails. In this model, the processing power of the secondary node(s) is mostly wasted in passive waiting for the primary node to fail.
-
-In an *active-active* model, no one node in the cluster acts as the primary node; all nodes in the cluster support the application. This application in this model runs on all the nodes, even splitting apart message processing between nodes to expedite processing across nodes.
+Mule uses an active-active model to cluster Mule runtime engines. The benefit of this model over an active-passive approach is that your application runs in all nodes, splitting apart message processing between the other nodes in your cluster expediting processing.
```

Comment:
"splitting message processing with the other nodes in your cluster, which expedites processing.